### DISCUSSION TITLE
Grafana LXC nor working (anymore)

### DISCUSSION THREAD

**Discussion by Besenreiter** [tteck/Proxmox#2191] [General]
Hello, I don't know if this is the right way to report, but the grafana-Install-script (not alpine) is not working:
 - Creating LXC Container...unable to create CT 104 - command 'lxc-usernsexec -m u:0:100000:65536 -m g:0:100000:65536 -- tar xpf - --zstd --totals --one-file-system -p --sparse --numeric-owner --acls --xattrs '--xattrs-include=user.*' '--xattrs-include=security.capability' '--warning=no-file-ignored' '--warning=no-xattr-write' -C /var/lib/lxc/104/rootfs --skip-old-files --anchored --exclude './dev/*'' failed: exit code 2
bash: line 163: exit: A problem occured while trying to create container.: numeric argument required

Thanks,
Ingo

**Comment by tteck** [comment]
Proxmox shell or SSH? 
Default settings or Advanced?

The script runs without any problems from this end. 
```
   ______           ____                 
  / ____/________ _/ __/___ _____  ____ _
 / / __/ ___/ __  / /_/ __  / __ \/ __  /
/ /_/ / /  / /_/ / __/ /_/ / / / / /_/ / 
\____/_/   \__,_/_/  \__,_/_/ /_/\__,_/  
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 100
Using Hostname: grafana
Using Disk Size: 2
Allocated Cores: 1
Allocated RAM: 512
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Grafana LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.5 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Grafana Repository
 ✓ Installed Grafana
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Grafana should be reachable by going to the following URL.
         http://192.168.86.5:3000 
```

**Comment by Besenreiter** [comment]
OK, stange. All other of your scripts are working for me. I tried several time, in the Proxmox-console (not via SSH). First I tried with "custom" setting, then to verify the problem with "default", same result. The alpine is working with custom and default.
The errormesage says something "in line 163", but your script has only 72 lines, so I didn't know where to look for.
Thanks, for your fast rrespose!
Ingo

**Reply by tteck** [reply]
What Linux distribution are you using, Debian?

**Comment by Besenreiter** [comment]
Yes, the Proxmox is on Debian 12.2
![grafik](https://github.com/tteck/Proxmox/assets/18124405/84312629-e1ab-4f5d-b4d7-dc0647289def)

**Reply by Besenreiter** [reply]
I'm new to Proxmox, so not sure what you are referring to. Are you talking about the Container-Tamplate?
![grafik](https://github.com/tteck/Proxmox/assets/18124405/93345503-7ae2-40f0-85e1-b6325121c80b)

**Comment by Besenreiter** [comment]
just saw there is a better errormessage under "tasks":
  Logical volume "vm-104-disk-0" created.
Creating filesystem with 524288 4k blocks and 131072 inodes
Filesystem UUID: 62912da1-d216-43a0-8d03-8b228d939cff
Superblock backups stored on blocks: 
	32768, 98304, 163840, 229376, 294912
extracting archive '/var/lib/vz/template/cache/debian-12-standard_12.2-1_amd64.tar.zst'
/*stdin*\ : Decoding error (36) : Restored data doesn't match checksum 
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
  Logical volume "vm-104-disk-0" successfully removed.

TASK ERROR: unable to create CT 104 - command 'lxc-usernsexec -m u:0:100000:65536 -m g:0:100000:65536 -- tar xpf - --zstd --totals --one-file-system -p --sparse --numeric-owner --acls --xattrs '--xattrs-include=user.*' '--xattrs-include=security.capability' '--warning=no-file-ignored' '--warning=no-xattr-write' -C /var/lib/lxc/104/rootfs --skip-old-files --anchored --exclude './dev/*'' failed: exit code 2

**Comment by Besenreiter** [comment]
Solved.
Sorry for having you bothered.
I tried to make new debian container. Even this was not working. When trieing to download the container template, I got a message the checksum from the local and the "to be downlowded" template don't match. removed the local, downloaded new, your script is working.
So, there must have been some kind of "degradation" in the template while waiting on my proxmox for usage. While it is nice to have it running now, it is not so nice to know that the data on this SSD is not that safe. I don't know what to do now with this info.
But anyway, thanks for your help and for your scripts,
Ingo

**Comment by rddaz2013** [comment]
TASK ERROR: unable to create CT 104 - command 'lxc-usernsexec -m u:0:100000:65536 -m g:0:100000:65536 -- tar xpf - --zstd --totals --one-file-system -p --sparse --numeric-owner --acls --xattrs '--xattrs-include=user.' '--xattrs-include=security.capability' '--warning=no-file-ignored' '--warning=no-xattr-write' -C /var/lib/lxc/104/rootfs --skip-old-files --anchored --exclude './dev/'' failed: exit code 2

same error with the Tdarr LXC if the debian version is used.... ubuntu version installs without error.
